examples/Resources/Private/Singles/Conditions.html

```diff
@@ -10,7 +10,12 @@
 	<f:else>Not touched</f:else>
 </f:if>
 
-Ternary expression: {checkTernary ? ternaryTrue : ternaryFalse}
+Ternary expression without then case: {ternaryTrue ? : ternaryFalse}
```

Comment:
I assume this also works with `?:` instead of `? :`? If yes, then we should prefer that since that's the common notation in PHP.

Reply:
Works both without and with any number and type of spaces between the parts.